src/main/java/org/launchcode/techjobs/persistent/controllers/SkillController.java

```diff
@@ -0,0 +1,64 @@
+package org.launchcode.techjobs.persistent.controllers;
+
+import org.launchcode.techjobs.persistent.models.Skill;
+import org.launchcode.techjobs.persistent.models.data.SkillRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.Errors;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.Optional;
+
+
+@Controller
+@RequestMapping("skill")
```

Comment:
```suggestion
@RequestMapping("skills")
```

This should be changed back to `skills` because the links in the template for Job's add.html expect `skills/add,html` in the URL path.